server/flaskr/database/handlers/organization_data_handler.py

```
@@ -37,7 +37,7 @@ def create_organization(cls, orgName: str) -> str | None:
                 )
 
                 with current_app.open_resource(
-                    os.path.join("database", "postgres", f"schema.sql")
+                    os.path.join("database", "sql", f"schema.sql")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)